tensorflow/python/ops/array_ops.py

```diff
@@ -4263,16 +4263,33 @@ def _with_nonzero_rank(data):
 
 @tf_export("repeat")
 def repeat(input, repeats, axis=None, name=None): # pylint: disable=redefined-builtin
-  """Repeat elements of an array
+  """Repeat elements of `input`
   Args:
-    input: A Tensor.
+    input: An `N`-dimensional Tensor.
     repeats: An 1-D `int` Tensor. The number of repetitions for each element.
-      repeats is broadcasted to fit the shape of the given axis
-    axis: An int. The axis along which to repeat values. By default, use the
-      flattened input array, and return a flat output array.
-    name: name of the op.
+      repeats is broadcasted to fit the shape of the given axis.
+      `len(repeats)` must equal `input.shape[axis]` if axis is not None.
+    axis: An int. The axis along which to repeat values. By default
+      (axis=None), use the flattened input array, and return a flat output
+      array.
+    name: A name for the operation.
   Returns:
     A Tensor which has the same shape as a, except along the given axis.
```

Comment:
nit: 

    A Tensor which has the same shape as `input`, except along the given axis.
                                         ^^^^^^^